pointmatcher/ErrorMinimizersImpl.h

```diff
@@ -95,30 +95,6 @@ struct ErrorMinimizersImpl
 		virtual T getOverlap() const;
 	};
 
-	struct PointToPlaneErrorMinimizer: public ErrorMinimizer
```

Comment:
Is it supported to create error minimizers from outside the library?
I think that moving this class away from this file implies that doing `new ErrorMinimizerImpl<float>::PointToPlaneErrorMinimizer()` does not work anymore. If we still want to support that from outside the library, we could add a typedef inside `ErrorMinimizerImpl<T>` that points to the new class.